src/profile-logic/processed-profile-versioning.js

```diff
@@ -276,5 +276,28 @@ const _upgraders = {
       delete thread.samples.frameNumber;
     }
   },
+  [6]: profile => {
+    for (const thread of profile.threads) {
```

Comment:
In the future we should explicitly add a comment of what the upgrade is doing. These blocks of code can be a bit mysterious otherwise.